### DISCUSSION TITLE
Java Tests not working

### DISCUSSION THREAD

**Discussion by Stefhermann** [nvim-java/nvim-java#311] [Q&A]
Im getting this error, 

Error executing Lua callback: ...nvim/lazy/nvim-java-test/lua/java-test/reports/junit.lua:23: attempt to index field 'result_parser' (a nil value)
stack traceback:
	...nvim/lazy/nvim-java-test/lua/java-test/reports/junit.lua:23: in function 'get_results'
	...nvim/lazy/nvim-java-test/lua/java-test/reports/junit.lua:28: in function 'show_report'
	...n/.local/share/nvim/lazy/nvim-java/lua/java/api/test.lua:64: in function <...n/.local/share/nvim/lazy/nvim-java/lua

When trying to run a test. I am using Astronvim with minimal changes to the normal configuration. Normal debugging works perfectly, just when I try to run Java Tests I get the error.

**Comment by wissamsu** [comment]
same issue did u find a fix?
